src/android/SplashScreen.java

```diff
@@ -91,7 +91,9 @@ protected void pluginInitialize() {
 
         firstShow = false;
         loadSpinner();
-        showSplashScreen(true);
+
+        boolean hideAfterDelay = preferences.getBoolean("AutoHideSplashScreen", true);
```

Comment:
The AutoHide preference is meant to hide without the use of a timer (e.g. hide on first paint after onload). Would be happy to accept a PR for this, but I don't think we should have a PR that just turns it off. For this, set a very large timeout.